go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/czerwonk/ping_exporter
+
+require (
+	github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc
+	github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf
+	github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973
+	github.com/czerwonk/ping_exporter v0.0.0-20181111181321-eed8f09ace8b
```

Comment:
I don't think we should require our own module ;)

Reply:
Ah yes, fixed!